src/plugins/data_table/data_table.js

```diff
@@ -377,7 +377,7 @@ var LocalDataTable = (function() {
       table.dataTable.find("thead th").each(function (index) {
         // here we use the text in the header to get the column config by title
         // there isn't a better way to do this currently
-        var title = this.outerText;
+        var title = this.textContent;
```

Comment:
I'm concerned about this change.

1) The original code assigns outerText to ``title``. ``outerText`` includes the contents of the tag, AND the surrounding tags themselves. 

An example can be found here: http://stackoverflow.com/questions/18481382/what-is-the-difference-between-innertext-and-outertext

https://msdn.microsoft.com/en-us/library/ms534311(v=vs.85).aspx

So with this change, whatever tags that surround the text will not get pulled into ``title``.

2) ```outerText``` is an IE specific tag, and it was only copied in the impl. of Opera, Chrome, and I *think* Safari. So the original code shouldn't have used it exclusively. However, ```textContent``` is not supported by IE (Maybe in MS Edge, but I'm not sure).

If we're sure that titles never have any HTML in them (see #1 above), the cross browser way to do this is: ```x.textContent || x.innerText```